openapi-integration-tests/src/test/java/io/ballerina/openapi/extension/build/ValidatorTests.java

```diff
@@ -204,4 +204,25 @@ public void typeMisMatchingInRecord() throws IOException {
             }
         }
     }
+
+    @Test(description = "Validator for resource path has root path '.'")
+    public void validatorWithRootPath() throws IOException {
+        List<String> buildArgs = new LinkedList<>();
+        buildArgs.add("project_8");
+        InputStream successful = TestUtil.executeOpenapiBuild(DISTRIBUTION_FILE_NAME, TEST_RESOURCE, buildArgs);
+        String msg = "ERROR [service.bal:(12:30,12:36)] Type mismatch with parameter 'id' for the" +
+                " method 'get' of the path '/'.In OpenAPI contract its type is 'string' and resources type is 'int'.";
```

Comment:
Shall we reword the error messages in a clearer way? (similar to what we did in https://github.com/ballerina-platform/openapi-tools/pull/893#discussion_r808152640)

Reply:
Yeah sure, Currently I'm rewording all the validator error messages with this issue https://github.com/ballerina-platform/openapi-tools/issues/910. Shall I address this via the fix from it? WDYT? 🙂